ibc-core/ics24-host/types/Cargo.toml

```diff
@@ -33,6 +35,9 @@ ibc-primitives = { workspace = true }
 parity-scale-codec = { workspace = true, optional = true }
 scale-info         = { workspace = true, optional = true }
 
+# cosmos dependencies
+tendermint-proto = { workspace = true }
```

Comment:
Let's shift away from `tendermint-proto`. I'm not sure if `ibc-proto-rs` exposes the `tendermint_proto::Error` we need. If it doesn't, I suggest we re-export it there, allowing us to remain dependent on `ibc-primitives` (since it exposes the necessary proto types).

Reply:
It looks like `ibc-proto-rs` currently does not export the error type. Does this merely involve adding `pub use tendermint_proto::Error;` to the root `lib.rs` file of `ibc-proto-rs`?

Reply:
Exactly. Maybe we go with `String` for now and add a TODO to revisit this once it's added to `ibc-proto-rs` and the new version is integrated.

Reply:
[b10ee71](https://github.com/cosmos/ibc-rs/pull/1325/commits/b10ee71322fef19bc456cc0f14aaa9d69bf9e3b5)

Reply:
Sounds like the `tendermint-proto` somehow is re-introduced here.